app/javascript/controllers/action_link_controller.js

```diff
@@ -0,0 +1,20 @@
+import { Controller } from "@hotwired/stimulus";
+
+export default class extends Controller {
+  static values = { required: { type: Number, default: 0 } };
+
+  #classes = [
```

Comment:
Please add a comment as to why the `#` is infront of the variable

Reply:
Added in 71d0b560f7ff2efa4c9fa50b28ecf755469fe42a